sdk/storage/azure-storage-files-datalake/src/datalake_file_system_client.cpp

```diff
@@ -194,11 +195,32 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     Models::CreateFileSystemResult ret;
     ret.ETag = std::move(result->ETag);
     ret.LastModified = std::move(result->LastModified);
+    ret.Created = true;
```

Comment:
Isn't this `true` already?

Reply:
Yes, but not relying on the default initialized value is a good practice as far as I'm concerned.